### DISCUSSION TITLE
Confirmation dialog with custom text

### DISCUSSION THREAD

**Discussion by FreddyX078** [marcokreeft87/room-card#318] [Ideas]
Hi,

Is it possible to use the confirmation dialog with a custom text? 
Something like: "Would you open the door?"

Last night my wife pressed the 'Open the door' button by accident and I'm glad we were home.
The confirmation dialog is already a step forward, but now it says: Are you sure you want to call-service?
I'm knowing my wife, next time see will press OK immediately when see reads: ... want ... service? 

Hope this is possible.

**Comment by marcokreeft87** [comment]
Look into browser-mod. That will allow you to make your own popups.

**Comment by FreddyX078** [comment]
Thanks for your reply and browser-mod looks a bit too much for this. 
I like to keep things simple and clean. 
Thats the reason I like and use room-card so much.

But I've modified room-card.js a bit and now I can enter custom text.

Things I've modified:
sed -i -e 's/{action:"more-info"}/{confirmation_text:"more-info"}/g' room-card/room-card.js
sed -i -e 's/"Are you sure you want to "+n.action+"?"/n.confirmation_text/' room-card/room-card.js
sed -i -e 's/"Are you sure you want to "+o.action+"?"/o.confirmation_text/' room-card/room-card.js

Add confirmation_text right under confirmation: true like this:
      confirmation: true
      confirmation_text: Weet je zeker dat je de deur wilt openen?

Because the minified js file, are there any negative side effects you now?
